### PR TITLE
Feature - Distributed DNS

### PR DESCRIPTION
**Pull request by philbrookes**
update manifests and bundle

upgrade controller-gen to 0.14.0 (#460)

dns: Bump DNS Operator version

propagate health check to DNS Records

### CODE REVIEW

File: api/v1alpha1/dnspolicy_types.go [hunk 1]
```diff
@@ -215,14 +213,14 @@ type DNSPolicyList struct {
...
-	Port                      *int                                      `json:"port,omitempty"`
-	Protocol                  *kuadrantdnsv1alpha1.HealthProtocol       `json:"protocol,omitempty"`
-	FailureThreshold          *int                                      `json:"failureThreshold,omitempty"`
-	AdditionalHeadersRef      *kuadrantdnsv1alpha1.AdditionalHeadersRef `json:"additionalHeadersRef,omitempty"`
-	ExpectedResponses         []int                                     `json:"expectedResponses,omitempty"`
-	AllowInsecureCertificates bool                                      `json:"allowInsecureCertificates,omitempty"`
-	Interval                  *metav1.Duration                          `json:"interval,omitempty"`
+	Endpoint                  string           `json:"endpoint,omitempty"`
+	Port                      *int             `json:"port,omitempty"`
+	Protocol                  *string          `json:"protocol,omitempty"`
```
Comment (maleck13) [inline api/v1alpha1/dnspolicy_types.go:218]: any specific reason to change these away from the original types?

File: bundle/manifests/kuadrant-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -262,6 +262,14 @@ spec:
           - patch
           - update
           - watch
+        - apiGroups:
```
Comment (maleck13) [inline bundle/manifests/kuadrant-operator.clusterserviceversion.yaml:265]: why do we need this permission now?

File: api/v1alpha1/dnspolicy_types.go (hunk 1 above)
Comment (maleck13) [reply api/v1alpha1/dnspolicy_types.go:218]: perhaps not compatible with external dns?

File: controllers/dnspolicy_controller_health_checks_test.go
Comment (maleck13) [inline controllers/dnspolicy_controller_health_checks_test.go:1]: do we not want to replace these with something else?

File: bundle/manifests/kuadrant-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (mikenairn) [reply bundle/manifests/kuadrant-operator.clusterserviceversion.yaml:265]: It's required for https://github.com/Kuadrant/kuadrant-operator/pull/550/files#diff-c1042ac0cb48fd02920d56c5a6426812656d90081386ed7dbefda245a0e0410cR157

**Comment by maleck13** [comment]
This is followed up by more work to improve test coverge https://github.com/Kuadrant/dns-operator/issues/44

**Pull request merged**